src/code/z_parameter.c

```diff
@@ -3789,7 +3789,7 @@ void Interface_Draw(GlobalContext* globalCtx) {
 
 void Interface_Update(GlobalContext* globalCtx) {
     static u8 D_80125B60 = 0;
-    static s16 D_80125B64 = 0;
+    static s16 sOldTimeIncrement = 0;
```

Comment:
```suggestion
    static s16 sPrevTimeIncrement = 0;
```
? old may actually be better, idk how often that value is refreshed

Reply:
it gets set right before suns starts speeding up time and is used to restore it when the song is done doing its thing. im not really sure one makes more sense over the other, they both convey that to me. but idk for sure

Reply:
using "prev" sounds better then, currently code uses "prev" all over